src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -17,13 +22,49 @@ private Drivetrain() {
      * @param leftSpeed  Speed of left side.
      * @param rightSpeed Speed of right side.
      */
-    public void tankDrive(double leftSpeed, double rightSpeed) {
```

Comment:
Please leave these as `leftSpeed` and `rightSpeed` or change the JavaDoc (comment) accordingly.